pkg/fleet/installer/service/apm_inject.go

```diff
@@ -216,6 +217,27 @@ func (a *apmInjectorInstaller) deleteLDPreloadConfigContent(ldSoPreload []byte)
 	return nil, fmt.Errorf("failed to remove %s from %s", launcherPreloadPath, ldSoPreloadPath)
 }
 
+// verifyLDPreloadFile verifies the LD preload file by:
+// 1. Parsing it
+// 2. Using the LD_PRELOAD env var, checking every single lib in the file to open a simple shell
+func (a *apmInjectorInstaller) verifyLDPreloadFile(path string) error {
+	file, err := os.ReadFile(path)
+	if err != nil {
+		return err
+	}
+	libs := strings.Fields(string(file))
+	for _, lib := range libs {
+		cmd := exec.Command("sh", "-c", "/bin/echo 1")
```

Comment:
can we remove the shell ? it adds a bunch of failure scenarios (simple example: their profile is broken)